requirements.txt

```diff
@@ -1,4 +1,4 @@
-acryl-datahub==1.1.0
-acryl-datahub-cloud==0.3.11
+acryl-datahub
```

Comment:
This is not a good idea. We should have pinned versions to ensure no weird incompatibilities between packages arise.